site/guide/customize-documentation-templates.qmd

```diff
@@ -83,40 +144,41 @@ a:hover {
 ```
 
-## Simplified template editing with **Edit**
+:::
 
-1. Log in to the {{< var vm_login >}}.
-
-{{< include _customize-documentation-templates-simplified.qmd >}}
-
-## Advanced YAML template editing with **Edit Raw**
+## Duplicate template
+To duplicate an existing template and start with version one of that new template:
```

Comment:
Trivial feedback: I would have expected it to say "version 1" instead of "version one". Seeing the number written out for a version seemed unnatural.

Reply:
Since this isn't an official named version, just the _concept_ of a version one, writing convention in most places/style guides dictate that numbers under 10 should be written as words unless referring to a specific named item. :)